learning-outcomes.md

```diff
@@ -0,0 +1,38 @@
+# Week 3 Learning Outcomes
+
+## Testing
+
+- [ ] **We can test the functionality of our app**
+- [ ] We can write tests to mimic the behaviour of a user performing different actions
+- [ ] We can check that passing a given input into our tests returns the expected output
+
+### Modular functions
+
+- [ ] **We can write testable, modular functions**
+- [ ] We can write functions that add DOM nodes
+- [ ] We can write functions that remove DOM nodes
+- [ ] We can write functions that modify DOM nodes
```

Comment:
Why the specific focus on DOM nodes in the modular functions section?

Reply:
I was thinking of the to-do list project. Perhaps these would be more suited to learning outcomes for the project though, rather than the week.

Reply:
I don't think those are bad outcomes for this week, I just don't know if they fit under "modular functions"

Reply:
I have changed the heading from 'Modular functions' to just 'Functions' to encompass all of these LOs

Reply:
We have a general "Javascript" heading in some weeks. Could we use that here and move the scope LO under it too?